docker-compose.yml

```diff
@@ -8,7 +8,6 @@ services:
     build:
       context: .
       dockerfile: Dockerfile
-    platform: linux/amd64
```

Comment:
My linux laptop doesn't like this line?

Reply:
interesting, i'm running on macos so probably why i didn't see it